02_bubble/marksomething/main_test.go

```diff
@@ -0,0 +1,96 @@
+package main
+
+import (
+	"bytes"
+	"fmt"
+	"reflect"
+	"testing"
+)
+
+func Test(t *testing.T) {
+
+	algorithms := []struct {
+		desc string
+		fn   func([]int) []int
+	}{
+		{"Bubble", bubble},
```

Comment:
instead of `desc` you could use `map[string]func(...)` see [Dave Cheney post](https://dave.cheney.net/2019/05/07/prefer-table-driven-tests) on table driven tests